CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 6.0.0
+* New classes to install Foreman plugins:
+    * foreman::plugin::azure to install Azure compute resource plugin
+    * foreman::plugin::expire_hosts to install expire hosts plugin
+    * foreman::plugin::host_extra_validator to install hostname validator plugin
+* New or changed parameters:
+    * Add server_port, server_ssl_port parameters to change Apache vhost ports
+    * Rename environment parameter to rails_env to fix compatibility with data
+      bindings
+* Other changes and fixes:
+    * node.rb: skip facts upload when facts file is missing, retrieves ENC
+      output anyway
+    * node.rb: improve logging for empty facts and failed fact uploads
+    * Change reports upload to use new config_reports API
+    * Change Yum GPG key URLs to HTTPS
+    * Fix missing default parameters for strict variables compatibility,
```

Comment:
#476 worked that around (at least for that one place), so requiring 4.6 should be ok, like it's stated in line 24?